billy-spain/src/main/java/com/premiumminds/billy/spain/services/builders/impl/ESSimpleInvoiceBuilderImpl.java

```diff
@@ -65,11 +64,11 @@ protected void validateInstance() throws BillyValidationException {
                 ESGenericInvoiceBuilderImpl.LOCALIZER.getString("field.clientType"));
         super.validateInstance();
 
-        if (i.getClientType() == CLIENTTYPE.CUSTOMER && i.getAmountWithTax().compareTo(new BigDecimal(1000)) >= 0) {
-            throw new BillySimpleInvoiceException("Amount > 1000 for customer simple invoice. Issue invoice");
+        if (i.getClientType() == CLIENTTYPE.CUSTOMER && i.getAmountWithTax().compareTo(new BigDecimal("1000")) >= 0) {
+            throw new InvalidAmountForDocumentTypeException("Amount > 1000 for customer simple invoice. Issue invoice");
```

Comment:
Shouldn't this value be 3000€ for ClientType.Customer?

Reply:
Indeed it is. I misread my notes and mixed up the values for PT and ES.